plugins/org/README-alpha.md

```diff
@@ -0,0 +1,462 @@
+
+## Routes
+
+The `org` plugin exposes a external route that can be used to configure route bindings.
+
+| Key            | Type           | Description                        |
+| -------------- | -------------- | ---------------------------------- |
+| `catalogIndex` | External route | A route ref to Catalog Index page. |
+
+As an example, here is an association between the external catalog index page and a regular route from another plugin:
+
+```yaml
+# app-config.yaml
+app:
+  routes:
+    bindings:
+      # example binding org and catalog index pages
+      org.catalogIndex: catalog.catalogIndex
+```
+
+Route binding is also possible through code. For more information, see [this](https://backstage.io/docs/frontend-system/architecture/routes#binding-external-route-references) documentation.
+
+## Extensions
+
+### Entity Cards
+
+The `org` plugin provide some entity cards you can enable to customize the Software Catalog entity page.
+
+> [!IMPORTANT]
+> The order in which cards are listed in the configuration file will determine the order in which they appear in overview cards and tab lists on entity pages.
+
+See a complete cards list below:
+
+#### Entity Group Profile Card
+
+This [entity card](https://github.com/backstage/backstage/blob/master/plugins/catalog-react/api-report-alpha.md) extension allows you to view, edit, or update groups metadata, such as team avatar, name, email, parent, and child groups.
+
+| Kind          | Namespace | Name            | Id                              | Example                                                                                   |
+| ------------- | --------- | --------------- | ------------------------------- | ----------------------------------------------------------------------------------------- |
+| `entity-card` | `org`     | `group-profile` | `entity-card:org/group-profile` | <img alt="Entity Group Profile Card" src="./OrgGroupProfileEntityCard.png" width="200" /> |
+
+##### Disable
+
+This card is disabled by default when you install the `org` plugin, but to ensure the card will always be disabled or enabled regardless of the extension's default definition, add the following configuration:
+
+```yaml
+# app-config.yaml
+# example disabling the org group profile entity card extension
+app:
+  extensions:
+    # this is the extension id and it follows the naming pattern bellow:
+    # <extension-kind>/<plugin-namespace>:<extension-name>
+    # use false as value for disabling the extension and true for enabling
+    - entity-card:org/group-profile: false
+    # or
+    # - entity-card:org/group-profile:
+    #   - config:
+    #       # set 'true' for enabling it again
+    #       disabled: true
+```
+
+##### Config
```

Comment:
This section (for all extensions everywhere) may be pretty suitable to have just as a single table.

Maybe have a general "how to set config for any extension" section somewhere else, for example in the architecture part, that we can link to first. Then a table with columns for like "Config key", "Default value", "Description" or so. No yaml should be necessary at that point I think

Reply:
I thought about this, but cards can have settings that may not be shared across all cards, so I ended up creating a settings subsection per card. Does it make sense?

Reply:
yeah but i meant a table with one row, in this section - instead of "free flowing text"

Reply:
Refactored 🙌🏻